README.md

```diff
@@ -13,15 +13,23 @@ Windows software for hosting locally connected USB devices to other machines, in
 This software requires Microsoft Windows 8 / Microsoft Windows Server 2012 or newer;
 it does not depend on any other software.
 
-1) Run the installer (.msi) from the <a href="https://github.com/dorssel/usbipd-win/releases/latest">latest release</a>
-   on the Windows machine where your USB device is connected.
+### Installation
 
-   Alternatively, use the Windows Package Manager:
-   <pre>
-   winget install usbipd
-   </pre>
+Run the installer (.msi) from the <a href="https://github.com/dorssel/usbipd-win/releases/latest">latest release</a> on the Windows machine where your USB device is connected. 
+
+Alternatively, use the Windows Package Manager:
+```
```

Comment:
Nice catch (maybe unknowingly...). I hadn't realized I used html tags, which is not proper markdown. There are more, but I will fix those later. Maybe I will add a markdown-linter check action to the build.